src/system/api/shared/include/gunwinput.h

```diff
@@ -0,0 +1,95 @@
+//
+//  gunwinput.h
+//  GunwOS
+//
+//  Created by Artur Danielewski on 07.03.2023.
+//
+
+#ifndef GUNWOS_GUNWINPUT_H
+#define GUNWOS_GUNWINPUT_H
+
+#include <types.h>
+
+/*
+    Event type code for keyboard event listener
+*/
+enum gnwKeyboardEventCode {
+    GKEC_KEY_DOWN,
+    GKEC_KEY_UP
+};
+_Static_assert(sizeof(enum gnwKeyboardEventCode) == sizeof(int_32), "Unexpected enum gnwKeyboardEventCode size");
```

Comment:
```suggestion
_Static_assert(sizeof(enum gnwKeyboardEventCode) == sizeof(int_32), "Expected size of enum gnwKeyboardEventCode is 32 bits");
```
or something like that

or_v2 provide _Static_assert_size(gnwKeyboardEventCode, sizeof(int32)) and generate the message in the macro